npm-react-codemod/test/pure-render-mixin-test2.output.js

```diff
@@ -1,8 +1,9 @@
-var React = require('react/addons');
+var React = require('react');
+var shallowCompare = require('react/addons/shallowCompare');
 
 var MyComponent = React.createClass({
   shouldComponentUpdate: function(nextProps, nextState) {
-    return React.addons.shallowCompare(this, nextProps, nextState);
```

Comment:
Additionally, I’m okay *not* changing these. I can back out these updates and only have docs updates. These tests will get a bit of noise due to the react/addons deprecation notice, but nothing will break at this time.